src/alfasim_sdk/_internal/constants.py

```diff
@@ -420,7 +420,18 @@ class PumpViscosityModel(Enum):
 
 class PumpThermalEfficiencyModel(Enum):
```

Comment:
Why is this class defined twice? :thinking:

Reply:
One of these class was removed. It was a mistake.